



>>> ter√ßa-feira, 24 de novembro de 2020 √†s 19:15 <<<

> <:emojicoffeeinfo:766020288885293097> Aviso!
> Alguns commandos foram atualizados no banco de dados!
> Request: {
    "aliases": [
        {
            "id": 1,
            "value": [
                "reportar"
            ]
        },
        {
            "id": 2,
            "value": [
                "sugestao"
            ]
        }
    ],
    "releases_notes": [
        {
            "id": 1,
            "value": {
                "1.0": null,
                "2.0": "Comando refeito, agora com um sistema de gest√£o de reports mais aprimorado. O usu√°rio que fez o report √© notificado quando o status do seu report muda, sabe se foi aceito ou rejeitado, e caso aceito, sabe quando foi solucionado!"
            }
        },
        {
            "id": 2,
            "value": {
                "1.0": null,
                "2.0": null
            }
        }
    ]
}
> Resposta: {
    "result": {
        "fieldCount": 0,
        "affectedRows": 2,
        "insertId": 0,
        "serverStatus": 2,
        "warningCount": 0,
        "message": "(Rows matched: 2  Changed: 0  Warnings: 0",
        "protocol41": true,
        "changedRows": 0
    },
    "sql": "UPDATE commands SET aliases = CASE id WHEN 1 THEN '[\"reportar\"]' WHEN 2 THEN '[\"sugestao\"]' ELSE aliases END, releases_notes = CASE id WHEN 1 THEN '{\"1.0\":null,\"2.0\":\"Comando refeito, agora com um sistema de gest√£o de reports mais aprimorado. O usu√°rio que fez o report √© notificado quando o status do seu report muda, sabe se foi aceito ou rejeitado, e caso aceito, sabe quando foi solucionado!\"}' WHEN 2 THEN '{\"1.0\":null,\"2.0\":null}' ELSE releases_notes END WHERE id IN (1, 2)",
    "commands": [
        {
            "id": 1,
            "name": "report",
            "aliases": [
                "reportar"
            ],
            "type": "Contato",
            "description": "Se voc√™ encontrou algum bug ou coisa que voc√™ acredita que n√£o esteja funcionando como deveria, basta usar o comando para que os devenvolvedores fiquem a par do problema e o resolva.",
            "how_to_use": "Digite _report no chat e passe como par√¢metro o seu report, em seguida envie a mensagem.",
            "cooldown": 10000,
            "example": "_report Encontrei um bug ao executar o comando ajuda, ao usar o comando o bot n√£o responde!",
            "example_url": "https://i.imgur.com/mAs7Ueq.png",
            "created_timestamp": 1598646020000,
            "updated_timestamp": 1605780157057,
            "version": "2.0",
            "releases_notes": {
                "1.0": null,
                "2.0": "Comando refeito, agora com um sistema de gest√£o de reports mais aprimorado. O usu√°rio que fez o report √© notificado quando o status do seu report muda, sabe se foi aceito ou rejeitado, e caso aceito, sabe quando foi solucionado!"
            },
            "times_limit": 2,
            "active": 1,
            "reason_inactivity": null
        },
        {
            "id": 2,
            "name": "sugerir",
            "aliases": [
                "sugestao"
            ],
            "type": "Contato",
            "description": "Se voc√™ gostaria de ver alguma nova funcionalidade no bot que ainda n√£o tenha, ou qualquer tipo de feature, basta usar o comando para que os devenvolvedores fiquem a par da sugest√£o e possa prontamente atender aos seus pedidos.",
            "how_to_use": "Digite _sugerir no chat e passe como par√¢metro a sua sugest√£o, em seguida envie a mensagem.",
            "cooldown": 10000,
            "example": "_sugerir Eu gostaria de um comando que me trouxesse caf√© na cama, PARA ONTEM!",
            "example_url": null,
            "created_timestamp": 1598663751000,
            "updated_timestamp": 1605780157057,
            "version": "2.0",
            "releases_notes": {
                "1.0": null,
                "2.0": null
            },
            "times_limit": 2,
            "active": 1,
            "reason_inactivity": null
        }
    ]
}



>>> ter√ßa-feira, 24 de novembro de 2020 √†s 19:15 <<<

> <:emojicoffeeinfo:766020288885293097> Aviso
> Ocorreu um erro ao enviar um novo log no canal cadastrado para logs.
> ID do canal: '757002422437543966'.
> Path: "C:\Users\Pichau\Documents\Bots\Coffe\src\functions\src\logger.js"
> Log: "> <:emojicoffeeinfo:766020288885293097> Aviso!
> Alguns commandos foram atualizados no banco de dados!
> Request: {
    "aliases": [
        {
            "id": 1,
            "value": [
                "reportar"
            ]
        },
        {
            "id": 2,
            "value": [
                "sugestao"
            ]
        }
    ],
    "releases_notes": [
        {
            "id": 1,
            "value": {
                "1.0": null,
                "2.0": "Comando refeito, agora com um sistema de gest√£o de reports mais aprimorado. O usu√°rio que fez o report √© notificado quando o status do seu report muda, sabe se foi aceito ou rejeitado, e caso aceito, sabe quando foi solucionado!"
            }
        },
        {
            "id": 2,
            "value": {
                "1.0": null,
                "2.0": null
            }
        }
    ]
}
> Resposta: {
    "result": {
        "fieldCount": 0,
        "affectedRows": 2,
        "insertId": 0,
        "serverStatus": 2,
        "warningCount": 0,
        "message": "(Rows matched: 2  Changed: 0  Warnings: 0",
        "protocol41": true,
        "changedRows": 0
    },
    "sql": "UPDATE commands SET aliases = CASE id WHEN 1 THEN '[\"reportar\"]' WHEN 2 THEN '[\"sugestao\"]' ELSE aliases END, releases_notes = CASE id WHEN 1 THEN '{\"1.0\":null,\"2.0\":\"Comando refeito, agora com um sistema de gest√£o de reports mais aprimorado. O usu√°rio que fez o report √© notificado quando o status do seu report muda, sabe se foi aceito ou rejeitado, e caso aceito, sabe quando foi solucionado!\"}' WHEN 2 THEN '{\"1.0\":null,\"2.0\":null}' ELSE releases_notes END WHERE id IN (1, 2)",
    "commands": [
        {
            "id": 1,
            "name": "report",
            "aliases": [
                "reportar"
            ],
            "type": "Contato",
            "description": "Se voc√™ encontrou algum bug ou coisa que voc√™ acredita que n√£o esteja funcionando como deveria, basta usar o comando para que os devenvolvedores fiquem a par do problema e o resolva.",
            "how_to_use": "Digite _report no chat e passe como par√¢metro o seu report, em seguida envie a mensagem.",
            "cooldown": 10000,
            "example": "_report Encontrei um bug ao executar o comando ajuda, ao usar o comando o bot n√£o responde!",
            "example_url": "https://i.imgur.com/mAs7Ueq.png",
            "created_timestamp": 1598646020000,
            "updated_timestamp": 1605780157057,
            "version": "2.0",
            "releases_notes": {
                "1.0": null,
                "2.0": "Comando refeito, agora com um sistema de gest√£o de reports mais aprimorado. O usu√°rio que fez o report √© notificado quando o status do seu report muda, sabe se foi aceito ou rejeitado, e caso aceito, sabe quando foi solucionado!"
            },
            "times_limit": 2,
            "active": 1,
            "reason_inactivity": null
        },
        {
            "id": 2,
            "name": "sugerir",
            "aliases": [
                "sugestao"
            ],
            "type": "Contato",
            "description": "Se voc√™ gostaria de ver alguma nova funcionalidade no bot que ainda n√£o tenha, ou qualquer tipo de feature, basta usar o comando para que os devenvolvedores fiquem a par da sugest√£o e possa prontamente atender aos seus pedidos.",
            "how_to_use": "Digite _sugerir no chat e passe como par√¢metro a sua sugest√£o, em seguida envie a mensagem.",
            "cooldown": 10000,
            "example": "_sugerir Eu gostaria de um comando que me trouxesse caf√© na cama, PARA ONTEM!",
            "example_url": null,
            "created_timestamp": 1598663751000,
            "updated_timestamp": 1605780157057,
            "version": "2.0",
            "releases_notes": {
                "1.0": null,
                "2.0": null
            },
            "times_limit": 2,
            "active": 1,
            "reason_inactivity": null
        }
    ]
}".
> Erro: "{
    "name": "DiscordAPIError",
    "message": "Invalid Form Body\ncontent: Must be 2000 or fewer in length.",
    "method": "post",
    "path": "/channels/757002422437543966/messages",
    "code": 50035,
    "httpStatus": 400
}"



>>> ter√ßa-feira, 24 de novembro de 2020 √†s 19:16 <<<

> <:emojicoffeeerro:766020289074167828> Erro!
> Houve um erro ao tentar criar um novo report na api!
> O report: "üëç"
> O ID do usu√°rio: "403925985847934976"
> Path: "C:\Users\Pichau\Documents\Bots\Coffe\src\commands\contact\report\src\createReport.js"
> O Erro: "{
    "message": "Request failed with status code 500",
    "name": "Error",
    "stack": "Error: Request failed with status code 500\n    at createError (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\createError.js:16:15)\n    at settle (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\settle.js:17:12)\n    at IncomingMessage.handleStreamEnd (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\adapters\\http.js:244:11)\n    at IncomingMessage.emit (events.js:327:22)\n    at endReadableNT (_stream_readable.js:1327:12)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)",
    "config": {
        "url": "/reports/create",
        "method": "put",
        "data": "{\"reports\":[{\"report\":\"üëç\",\"created_by\":\"403925985847934976\"}]}",
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Content-Type": "application/json;charset=utf-8",
            "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjczOTMwOTMwNTU4ODc0NDI2MyIsImlhdCI6MTYwNjI1NjE5MH0.zO8NqGrapWUs2BMref-JbuX0X2ZpCaj-r_ahzxM0uzw",
            "User-Agent": "axios/0.20.0",
            "Content-Length": 65
        },
        "baseURL": "http://localhost:3001/api",
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1
    }
}
{
    "error": "Error when creating",
    "sqlError": {
        "code": "ER_TRUNCATED_WRONG_VALUE_FOR_FIELD",
        "errno": 1366,
        "sqlMessage": "Incorrect string value: '\\xF0\\x9F\\x91\\x8D' for column 'report' at row 1",
        "sqlState": "HY000",
        "index": 0,
        "sql": "INSERT INTO reports (report, created_by, created_timestamp, status) VALUES ('üëç', '403925985847934976', '1606256190138', 'EM ANALISE')"
    }
}"



>>> ter√ßa-feira, 24 de novembro de 2020 √†s 19:25 <<<

> <:emojicoffeeerro:766020289074167828> Erro!
> Houve um erro ao tentar criar um novo report na api!
> O report: "üëç"
> O ID do usu√°rio: "403925985847934976"
> Path: "C:\Users\Pichau\Documents\Bots\Coffe\src\commands\contact\report\src\createReport.js"
> O Erro: "{
    "message": "Request failed with status code 500",
    "name": "Error",
    "stack": "Error: Request failed with status code 500\n    at createError (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\createError.js:16:15)\n    at settle (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\settle.js:17:12)\n    at IncomingMessage.handleStreamEnd (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\adapters\\http.js:244:11)\n    at IncomingMessage.emit (events.js:327:22)\n    at endReadableNT (_stream_readable.js:1327:12)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)",
    "config": {
        "url": "/reports/create",
        "method": "put",
        "data": "{\"reports\":[{\"report\":\"üëç\",\"created_by\":\"403925985847934976\"}]}",
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Content-Type": "application/json;charset=utf-8",
            "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjczOTMwOTMwNTU4ODc0NDI2MyIsImlhdCI6MTYwNjI1NjcxOX0.TZFvv9QL44azEtlnRVaGfu6l_S_QvtUAFEwbDlNVmRQ",
            "User-Agent": "axios/0.20.0",
            "Content-Length": 65
        },
        "baseURL": "http://localhost:3001/api",
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1
    }
}
{
    "error": "Error when creating",
    "sqlError": {
        "code": "ER_TRUNCATED_WRONG_VALUE_FOR_FIELD",
        "errno": 1366,
        "sqlMessage": "Incorrect string value: '\\xF0\\x9F\\x91\\x8D' for column 'report' at row 1",
        "sqlState": "HY000",
        "index": 0,
        "sql": "INSERT INTO reports (report, created_by, created_timestamp, status) VALUES ('üëç', '403925985847934976', '1606256719837', 'EM ANALISE')"
    }
}"



>>> ter√ßa-feira, 24 de novembro de 2020 √†s 19:26 <<<

> <:emojicoffeeerro:766020289074167828> Erro!
> Houve um erro ao tentar criar um novo report na api!
> O report: "üëç"
> O ID do usu√°rio: "403925985847934976"
> Path: "C:\Users\Pichau\Documents\Bots\Coffe\src\commands\contact\report\src\createReport.js"
> O Erro: "{
    "message": "Request failed with status code 500",
    "name": "Error",
    "stack": "Error: Request failed with status code 500\n    at createError (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\createError.js:16:15)\n    at settle (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\settle.js:17:12)\n    at IncomingMessage.handleStreamEnd (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\adapters\\http.js:244:11)\n    at IncomingMessage.emit (events.js:327:22)\n    at endReadableNT (_stream_readable.js:1327:12)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)",
    "config": {
        "url": "/reports/create",
        "method": "put",
        "data": "{\"reports\":[{\"report\":\"üëç\",\"created_by\":\"403925985847934976\"}]}",
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Content-Type": "application/json;charset=utf-8",
            "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjczOTMwOTMwNTU4ODc0NDI2MyIsImlhdCI6MTYwNjI1NjgxOH0.CN70bgJNtAWefisgyN8hfiYp6qwHhmc2r2k3ZWWiq9g",
            "User-Agent": "axios/0.20.0",
            "Content-Length": 65
        },
        "baseURL": "http://localhost:3001/api",
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1
    }
}
{
    "error": "Error when creating",
    "sqlError": {
        "code": "ER_TRUNCATED_WRONG_VALUE_FOR_FIELD",
        "errno": 1366,
        "sqlMessage": "Incorrect string value: '\\xF0\\x9F\\x91\\x8D' for column 'report' at row 1",
        "sqlState": "HY000",
        "index": 0,
        "sql": "INSERT INTO reports (report, created_by, created_timestamp, status) VALUES ('üëç', '403925985847934976', '1606256818964', 'EM ANALISE')"
    }
}"



>>> ter√ßa-feira, 24 de novembro de 2020 √†s 20:16 <<<

> <:emojicoffeeerro:766020289074167828> Erro!
> Houve um erro ao fazer uma requisi√ß√£o dos members para a API!
> Path: "C:\Users\Pichau\Documents\Bots\Coffe\src\controllers\scoreController\index.js"
> Erro: "{
    "message": "connect ECONNREFUSED 127.0.0.1:3001",
    "name": "Error",
    "stack": "Error: connect ECONNREFUSED 127.0.0.1:3001\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16)",
    "config": {
        "url": "/members",
        "method": "get",
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjczOTMwOTMwNTU4ODc0NDI2MyIsImlhdCI6MTYwNjI1OTc5M30.CtxSQePc8Fxu7aVUxb67qyryj5r2_Awt1aWji8zFYkw",
            "User-Agent": "axios/0.20.0"
        },
        "baseURL": "http://localhost:3001/api",
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1
    },
    "code": "ECONNREFUSED"
}
"



>>> ter√ßa-feira, 24 de novembro de 2020 √†s 20:22 <<<

> <:emojicoffeeerro:766020289074167828> Erro!
> Houve um erro ao tentar criar um novo report na api!
> O report: "üò©"
> O ID do usu√°rio: "403925985847934976"
> Path: "C:\Users\Pichau\Documents\Bots\Coffe\src\commands\contact\report\src\createReport.js"
> O Erro: "{
    "message": "Request failed with status code 500",
    "name": "Error",
    "stack": "Error: Request failed with status code 500\n    at createError (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\createError.js:16:15)\n    at settle (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\settle.js:17:12)\n    at IncomingMessage.handleStreamEnd (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\adapters\\http.js:244:11)\n    at IncomingMessage.emit (events.js:327:22)\n    at endReadableNT (_stream_readable.js:1327:12)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)",
    "config": {
        "url": "/reports/create",
        "method": "put",
        "data": "{\"reports\":[{\"report\":\"üò©\",\"created_by\":\"403925985847934976\"}]}",
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Content-Type": "application/json;charset=utf-8",
            "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjczOTMwOTMwNTU4ODc0NDI2MyIsImlhdCI6MTYwNjI2MDE1Nn0.gtfHD6VcadJP8GYu8rYWMMDYR6GXess4uD2CvCqX8zc",
            "User-Agent": "axios/0.20.0",
            "Content-Length": 65
        },
        "baseURL": "http://localhost:3001/api",
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1
    }
}
{
    "error": "Error when creating",
    "sqlError": {
        "code": "ER_TRUNCATED_WRONG_VALUE_FOR_FIELD",
        "errno": 1366,
        "sqlMessage": "Incorrect string value: '\\xF0\\x9F\\x98\\xA9' for column 'report' at row 1",
        "sqlState": "HY000",
        "index": 0,
        "sql": "INSERT INTO reports (report, created_by, created_timestamp, status) VALUES ('üò©', '403925985847934976', '1606260156997', 'EM ANALISE')"
    }
}"



>>> ter√ßa-feira, 24 de novembro de 2020 √†s 20:23 <<<

> <:emojicoffeeerro:766020289074167828> Erro!
> Houve um erro ao tentar criar um novo report na api!
> O report: "üò©"
> O ID do usu√°rio: "403925985847934976"
> Path: "C:\Users\Pichau\Documents\Bots\Coffe\src\commands\contact\report\src\createReport.js"
> O Erro: "{
    "message": "Request failed with status code 500",
    "name": "Error",
    "stack": "Error: Request failed with status code 500\n    at createError (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\createError.js:16:15)\n    at settle (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\settle.js:17:12)\n    at IncomingMessage.handleStreamEnd (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\adapters\\http.js:244:11)\n    at IncomingMessage.emit (events.js:327:22)\n    at endReadableNT (_stream_readable.js:1327:12)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)",
    "config": {
        "url": "/reports/create",
        "method": "put",
        "data": "{\"reports\":[{\"report\":\"üò©\",\"created_by\":\"403925985847934976\"}]}",
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Content-Type": "application/json;charset=utf-8",
            "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjczOTMwOTMwNTU4ODc0NDI2MyIsImlhdCI6MTYwNjI2MDIxMn0.WPKS-PsoDfd1Z1pTZHOp6ilBBhI6ihNd1ttC5pdXH-w",
            "User-Agent": "axios/0.20.0",
            "Content-Length": 65
        },
        "baseURL": "http://localhost:3001/api",
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1
    }
}
{
    "error": "Error when creating",
    "sqlError": {
        "code": "ER_TRUNCATED_WRONG_VALUE_FOR_FIELD",
        "errno": 1366,
        "sqlMessage": "Incorrect string value: '\\xF0\\x9F\\x98\\xA9' for column 'report' at row 1",
        "sqlState": "HY000",
        "index": 0,
        "sql": "INSERT INTO reports (report, created_by, created_timestamp, status) VALUES ('üò©', '403925985847934976', '1606260212202', 'EM ANALISE')"
    }
}"



>>> ter√ßa-feira, 24 de novembro de 2020 √†s 20:24 <<<

> <:emojicoffeeerro:766020289074167828> Erro!
> Houve um erro ao tentar criar um novo report na api!
> O report: "üò©"
> O ID do usu√°rio: "403925985847934976"
> Path: "C:\Users\Pichau\Documents\Bots\Coffe\src\commands\contact\report\src\createReport.js"
> O Erro: "{
    "message": "Request failed with status code 500",
    "name": "Error",
    "stack": "Error: Request failed with status code 500\n    at createError (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\createError.js:16:15)\n    at settle (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\settle.js:17:12)\n    at IncomingMessage.handleStreamEnd (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\adapters\\http.js:244:11)\n    at IncomingMessage.emit (events.js:327:22)\n    at endReadableNT (_stream_readable.js:1327:12)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)",
    "config": {
        "url": "/reports/create",
        "method": "put",
        "data": "{\"reports\":[{\"report\":\"üò©\",\"created_by\":\"403925985847934976\"}]}",
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Content-Type": "application/json;charset=utf-8",
            "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjczOTMwOTMwNTU4ODc0NDI2MyIsImlhdCI6MTYwNjI2MDI5OH0.M-segslcuhB5sW7eIRBWbQp11InEm5Ou95p1zPGxhgc",
            "User-Agent": "axios/0.20.0",
            "Content-Length": 65
        },
        "baseURL": "http://localhost:3001/api",
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1
    }
}
{
    "error": "Error when creating",
    "sqlError": {
        "code": "ER_TRUNCATED_WRONG_VALUE_FOR_FIELD",
        "errno": 1366,
        "sqlMessage": "Incorrect string value: '\\xF0\\x9F\\x98\\xA9' for column 'report' at row 1",
        "sqlState": "HY000",
        "index": 0,
        "sql": "INSERT INTO reports (report, created_by, created_timestamp, status) VALUES ('üò©', '403925985847934976', '1606260298451', 'EM ANALISE')"
    }
}"



>>> ter√ßa-feira, 24 de novembro de 2020 √†s 20:26 <<<

> <:emojicoffeeerro:766020289074167828> Erro!
> Houve um erro ao tentar criar um novo report na api!
> O report: "üò©"
> O ID do usu√°rio: "403925985847934976"
> Path: "C:\Users\Pichau\Documents\Bots\Coffe\src\commands\contact\report\src\createReport.js"
> O Erro: "{
    "message": "Request failed with status code 500",
    "name": "Error",
    "stack": "Error: Request failed with status code 500\n    at createError (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\createError.js:16:15)\n    at settle (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\settle.js:17:12)\n    at IncomingMessage.handleStreamEnd (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\adapters\\http.js:244:11)\n    at IncomingMessage.emit (events.js:327:22)\n    at endReadableNT (_stream_readable.js:1327:12)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)",
    "config": {
        "url": "/reports/create",
        "method": "put",
        "data": "{\"reports\":[{\"report\":\"üò©\",\"created_by\":\"403925985847934976\"}]}",
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Content-Type": "application/json;charset=utf-8",
            "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjczOTMwOTMwNTU4ODc0NDI2MyIsImlhdCI6MTYwNjI2MDQwNH0.almqjQ0RX4JNbKU5eDx4y68klWOjFw1mLyBTwmHnr0g",
            "User-Agent": "axios/0.20.0",
            "Content-Length": 65
        },
        "baseURL": "http://localhost:3001/api",
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1
    }
}
{
    "error": "Error when creating",
    "sqlError": {
        "code": "ER_TRUNCATED_WRONG_VALUE_FOR_FIELD",
        "errno": 1366,
        "sqlMessage": "Incorrect string value: '\\xF0\\x9F\\x98\\xA9' for column 'report' at row 1",
        "sqlState": "HY000",
        "index": 0,
        "sql": "INSERT INTO reports (report, created_by, created_timestamp, status) VALUES ('üò©', '403925985847934976', '1606260404819', 'EM ANALISE')"
    }
}"



>>> ter√ßa-feira, 24 de novembro de 2020 √†s 20:27 <<<

> <:emojicoffeeerro:766020289074167828> Erro!
> Houve um erro ao buscar o prefix de um servidor na API!
> Servidor: "Discord Bot Development" `672933215836569610`
> Path: "C:\Users\Pichau\Documents\Bots\Coffe\src\functions\src\prefix.js"
> Erro: "{
    "message": "connect ECONNREFUSED 127.0.0.1:3001",
    "name": "Error",
    "stack": "Error: connect ECONNREFUSED 127.0.0.1:3001\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16)",
    "config": {
        "url": "/servers/672933215836569610",
        "method": "get",
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjczOTMwOTMwNTU4ODc0NDI2MyIsImlhdCI6MTYwNjI2MDQ0Nn0.eOpRZ-E7KWk8zHOHjs9nBmIlvvAKpGweTzY9BIksHcI",
            "User-Agent": "axios/0.20.0"
        },
        "baseURL": "http://localhost:3001/api",
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1
    },
    "code": "ECONNREFUSED"
}
"



>>> ter√ßa-feira, 24 de novembro de 2020 √†s 21:21 <<<

> <:emojicoffeeerro:766020289074167828> Erro!
> Houve um erro ao tentar criar um novo suggestion na api!
> O suggestion: "dx o coffee on pra mim usa o dailyüòé"
> O ID do usu√°rio: "477534823011844120"
> Path: "C:\Users\Pichau\Documents\Bots\Coffe\src\commands\contact\suggestion\src\createSuggestion.js"
> O Erro: "{
    "message": "Request failed with status code 500",
    "name": "Error",
    "stack": "Error: Request failed with status code 500\n    at createError (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\createError.js:16:15)\n    at settle (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\settle.js:17:12)\n    at IncomingMessage.handleStreamEnd (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\adapters\\http.js:244:11)\n    at IncomingMessage.emit (events.js:327:22)\n    at endReadableNT (_stream_readable.js:1327:12)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)",
    "config": {
        "url": "/suggestions/create",
        "method": "put",
        "data": "{\"suggestions\":[{\"suggestion\":\"dx o coffee on pra mim usa o dailyüòé\",\"created_by\":\"477534823011844120\"}]}",
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Content-Type": "application/json;charset=utf-8",
            "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjczOTMwOTMwNTU4ODc0NDI2MyIsImlhdCI6MTYwNjI2MzY5NX0.k35WrQ2KEpozsaP-ZvlaQAg-LB3troeon_KYkXPCPoM",
            "User-Agent": "axios/0.20.0",
            "Content-Length": 107
        },
        "baseURL": "http://localhost:3001/api",
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1
    }
}
{
    "error": "Error when creating",
    "sqlError": {
        "code": "ER_TRUNCATED_WRONG_VALUE_FOR_FIELD",
        "errno": 1366,
        "sqlMessage": "Incorrect string value: '\\xF0\\x9F\\x98\\x8E' for column 'suggestion' at row 1",
        "sqlState": "HY000",
        "index": 0,
        "sql": "INSERT INTO suggestions (suggestion, created_by, created_timestamp, status) VALUES ('dx o coffee on pra mim usa o dailyüòé', '477534823011844120', '1606263695299', 'EM ANALISE')"
    }
}"